widget/balance/src/main/java/com/ivy/widget/balance/WalletBalanceWidget.kt

```diff
@@ -35,6 +35,11 @@ import javax.inject.Inject
 import kotlin.math.abs
 
 const val THOUSAND = 1000
+const val APP_LOCKED = "appLocked"
+const val BALANCE_V2 = "balance_v2"
```

Comment:
Remove the `_V2` from the constant name. The make the code simpler you can in the same file create:
```kotlin
object DataStoreKeys {
   const val AppLocked = "..."
   const val Balance = "balance_v2"
}
```

Reply:
Updated